src/parsers/locale-messagesjson.js

```diff
@@ -153,6 +159,29 @@ export default class LocaleMessagesJSONParser extends JSONParser {
         );
       }
 
+      if (this.addonMetadata && typeof this.addonMetadata.name === 'string') {
+        const messageNameRegexp = new RegExp(MANFIEST_MESSAGE_NAME_REGEXP, 'g');
+        const match = messageNameRegexp.exec(this.addonMetadata.name);
```

Comment:
This match should probably be found outside of the loop this is in.

Reply:
I really missed this one. fixed!